packages/react-lowcode/src/codegen/cli/index.ts

```diff
@@ -58,7 +58,7 @@ class LocalCodegenCli{
         
         const printer = ts.createPrinter({ newLine: ts.NewLineKind.LineFeed })
   
-        let sourceCode = printer.printList(ts.ListFormat.MultiLine, factory.createNodeArray([...page.imports, page.functionDeclaration]), sourceFile)
+        let sourceCode = printer.printList(ts.ListFormat.MultiLine, factory.createNodeArray([...page!.imports, page!.functionDeclaration]), sourceFile)
```

Comment:
`!.` is always suspicious. Isn't there cleaner way?

Reply:
It is because we are on half way to refactor generator according to designed architecture. Lot of things inside are coded pretty ugly. We need to refactor whole generator after milestone 'generate whole app' will be done.